### DISCUSSION TITLE
OBS does not support the latest version of macOS (Sonoma 14.2.1)

### DISCUSSION THREAD

**Discussion by SniperMango** [obsproject/obs-studio#10093] [Development Questions]
To OBS Studios,

macOS has updated to 14.2.1 Sonoma and OBS has outdated code to support it. Because of this, OBS won’t begin to run and will require an updated version for use.

I would be very appreciative if there is a new version released that supports my macOS version that will allow me to stream and create YouTube content again, as well as screen sharing gameplay to my friends on Discord.

Kind Regards, Mango

**Comment by tytan652** [comment]
OBS Studio 30 completely supports macOS 14 including the virtual camera feature.

**Reply by SniperMango** [reply]
@tytan652 On Intel or Silicon macs?

**Reply by tytan652** [reply]
macOS still support both, so both.

**Reply by feetstv** [reply]
@SniperMango Head over to the OBS Studio Community [Discord](https://obsproject.com/discord) or [Forums](https://obsproject.com/forum/) for support.

**Comment by JustAnEric** [comment]
I am having the same issue too.

**Reply by feetstv** [reply]
Head over to the OBS Studio Community [Discord](https://obsproject.com/discord) or [Forums](https://obsproject.com/forum/) for support.
